core/src/main/python/wlsdeploy/tool/discover/topology_discoverer.py

```diff
@@ -768,7 +768,7 @@ def _dynamic_server(self, server, location):
                     _logger.fine('WLSDPLY-06613', server, class_name=_class_name, method_name=_method_name)
                     self._wlst_helper.cd(wlst_path)
                     attr_name = self._aliases.get_wlst_attribute_name(cluster_location,
-                                                                      model_constants.DYNAMIC_CLUSTER_SIZE)
+                                                                          model_constants.MAX_DYNAMIC_CLUSTER_SIZE)
```

Comment:
It should work but the MaximumDynamicServerCount is deprecated  

@Deprecated
int getMaximumDynamicServerCount()
Deprecated. 12.2.1.0.0 Use getDynamicClusterSize()
The maximum number of server instances allowed to be created. This value should be set to the number of servers expected to be in the cluster. The specified number of servers are dynamically added to the configuration at runtime and associated ServerLifeCycleRuntimeMBeans are created.
Returns:
maximum number of dynamic servers to be created
Changes to this property are dynamic and thus take effect immediately without restarting the server or partition.
Maximum Value:
800
Minimum Value:
0

Reply:
changed